src/test/java/domain/LadderGameTest.java

```diff
@@ -0,0 +1,37 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class LadderGameTest {
+
+    @DisplayName("이름이 입력되면 알맞은 결과를 반환해야한다.")
+    @ParameterizedTest
+    @CsvSource(value = {"pobi:포비결과", "glen:글렌결과", "vero:베로결과"}, delimiter = ':')
+    void test(String name, String expectedResult) {
+        //given
+        LadderGame ladderGame = createLadderGame();
```

Comment:
`@ParameterizedTest` 를 사용할 때 DisplayName 을 확인하려면 아래와같이 사용해볼 수 있을것같아요

```suggestion
@DisplayName("이름이 입력되면 알맞은 결과를 반환해야한다.")
    @ParameterizedTest(name = "{displayName} name: {0}, expectedResult: {1}")
    @CsvSource(value = {"pobi:포비결과", "glen:글렌결과", "vero:베로결과"}, delimiter = ':')
    void test(String name, String expectedResult) {
```

Reply:
꿀팁 감사합니다 테스트가 더욱 보기 편해질 것 같습니다.